hnn_core/tests/test_optimization.py

```diff
@@ -116,26 +116,37 @@ def test_optimize_evoked():
         net_empty = net_offset.copy()
         del net_empty.external_drives['evprox1']
         del net_empty.external_drives['evdist1']
-        net_opt = optimize_evoked(net_empty, tstop=tstop, n_trials=n_trials,
-                                  target_dpl=dpl_orig, initial_dpl=dpl_offset,
-                                  maxiter=10)
+        net_opt, rmse = optimize_evoked(net_empty, tstop=tstop,
+                                        n_trials=n_trials, target_dpl=dpl_orig,
+                                        initial_dpl=dpl_offset, maxiter=10,
+                                        return_rmse=True)
 
     with pytest.raises(ValueError, match='The drives selected to be optimized '
                        'are not evoked drives'):
         net_test_bursty = net_offset.copy()
         which_drives = ['bursty1']
-        net_opt = optimize_evoked(net_test_bursty, tstop=tstop,
-                                  n_trials=n_trials, target_dpl=dpl_orig,
-                                  initial_dpl=dpl_offset,
-                                  which_drives=which_drives, maxiter=10)
+        net_opt, rmse = optimize_evoked(net_test_bursty, tstop=tstop,
+                                        n_trials=n_trials, target_dpl=dpl_orig,
+                                        initial_dpl=dpl_offset,
+                                        which_drives=which_drives, maxiter=10,
+                                        return_rmse=True)
```

Comment:
Since the new `return_rmse` feature is not actually being tested here, I'd recommend leaving these tests in their original state for simplicity.